core/storage/blog_post/gae_models.py

```diff
@@ -0,0 +1,454 @@
+# coding: utf-8
```

Comment:
Will discuss it with sean and prayush and update accordingly.

Reply:
We will keep it -'blog_post' as even if we have any other model added to it, they will be related to the blog post and following the convention as the main model is for blog post, the directory name will be blog_post.
 /cc @iamprayush

Reply:
I actually thought it should be "blog" too. E.g. consider the case if you have models in the future that are related to the blog as a whole, like the list of allowed authors or the list of different blogs (maybe there's an engineering blog and an outreach blog), or stuff like that. I think it's too early to make the call that this will be specific to blog-post stuff only.

Reply:
Done. Renamed the directory to blog. And accordingly changed at other places as well.